pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["setuptools >= 61", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "league-rpc-linux"
+dynamic = ["version"]
+requires-python = ">= 3.8"
+dependencies = [
+    "pypresence >= 4.3.0",
+    "psutil >= 5.9.6",
+    "requests >= 2.31.0"
+]
+
+[tool.setuptools]
+packages = ["league_rpc_linux", "league_rpc_linux.processes"]
```

Comment:
is league_rpc_linux.processes really nessesary here?

Reply:
Yes because in Python each subdirectory counts as subpackage and using this approach need to list everything explicitly. But now I switched to automatic inclusion so everything gets added and no need to explicitly specify them.